### PR TITLE
DEV-29: Forgot-Password-Page

### PR DESCRIPTION
**Pull request by AntonSydorovych**
## Ticket
Link to ticket:  https://bsalo.atlassian.net/browse/DEV-29

## Description
*Please provide a small summary of your changes and how they resolved the issues in the ticket:*

- Created a Forgot Password page
- Fixed Sign In page

Screenshot_6
Screenshot_7

(prefer animated gif)

## Checklist

- [x] Did you test it?
- [x] Did you add tests?
- [x] Did you add to the storybook?

### CODE REVIEW

**Comment by AntonSydorovych** [comment]
fix facebook button

**Comment by AntonSydorovych** [comment]
fix facebook button

File: src/pages/ForgotPassword/ForgotPassword.tsx [hunk 1]
```diff
@@ -1,5 +1,85 @@
...
+                    maxLength: 50,
+                    pattern: EMAIL_PATTERN,
+                  })}
+                />
+                <FormErrorMessage message={errors.email?.message} />
+              </Form.Item>
+
+              <Form.Item>
+                <Button
+                  className="_reset-password"
```
Comment (dev1B) [inline src/pages/ForgotPassword/ForgotPassword.tsx:52]: _

File: src/pages/ForgotPassword/ForgotPassword.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          font-size: 20px;
+          color: rgb(156, 156, 156);
+        }
+      }
+
+      input,
+      button {
+        margin: 5px 0px;
+
+        &._reset-password {
```
Comment (dev1B) [inline src/pages/ForgotPassword/ForgotPassword.scss:41]: _

File: src/pages/ForgotPassword/ForgotPassword.tsx [hunk 3]
```diff
@@ -1,5 +1,85 @@
...
+    <div className="forgot-password">
+      <Row justify="center">
+        <Col className="img-block" xs={0} sm={0} md={10} lg={10} xl={9} xxl={8}>
+          <img src="https://i.imgflip.com/5jwx15.jpg" alt="Welcome on portal" />
+        </Col>
+
+        <Col className="form-block" xs={20} sm={22} md={10} lg={10} xl={9} xxl={8}>
+          <Col span={18}>
+            <Form onFinish={handleSubmit(onSubmit)}>
+              <Title className="title-form" level={2}>
```
Comment (dev1B) [inline src/pages/ForgotPassword/ForgotPassword.tsx:31]: antd -> title + paragraph

Comment (dev1B) [reply src/pages/ForgotPassword/ForgotPassword.tsx:31]: https://ant.design/components/typography/#components-typography-demo-title

File: src/stories/ForgotPassword.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import ForgotPassword from '../pages/ForgotPassword/ForgotPassword';
+
+export default {
+  title: 'Reset Password/ForgotPassword',
```
Comment (dev1B) [inline src/stories/ForgotPassword.stories.tsx:6]: Pages/Forgot....

File: src/stories/ForgotPassword.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  component: ForgotPassword,
+  argTypes: { backgroundColor: { control: 'color' } },
```
Comment (dev1B) [inline src/stories/ForgotPassword.stories.tsx:8]: безтолкове 
`argTypes: { backgroundColor: { control: 'color' } },`

File: src/pages/ForgotPassword/ForgotPassword.tsx [hunk 6]
```diff
@@ -1,5 +1,71 @@
...
+                  htmlType="submit"
+                  shape="round"
+                >
+                  Reset Password
+                </Button>
+              </Form.Item>
+
+              <hr className="line" />
+
+              <div className="footer">
```
Comment (dev1B) [inline src/pages/ForgotPassword/ForgotPassword.tsx:60]: Може бути nav

**Review by dev1B — CHANGES_REQUESTED** [review]

**Comment by dev1B** [comment]
@AntonSydorovych fix merge conflict to move forward

**Review by dev1B — APPROVED** [review]

**Pull request merged**